lunatrace/bsl/frontend/src/api/graphql/getSidebarInfo.graphql

```diff
@@ -27,4 +27,10 @@ query GetSidebarInfo {
             }
         }
     }
+
+    users(where: $users_filter) {
```

Comment:
As discussed we are moving this to its own API call and only calling it from within the SideBar.tsx component.